dftimewolf/lib/collectors/grr_hunt.py

```diff
@@ -247,20 +259,34 @@ def _extract_hunt_results(self, output_file_path):
     # Extract items from archive by host for processing
     collection_paths = []
     clients = set()
+    client_id_fqdn = {}
+    hunt_dir = None
     try:
       with zipfile.ZipFile(output_file_path) as archive:
         items = archive.infolist()
         for f in items:
+
+          if not hunt_dir:
+            hunt_dir = f.filename.split('/')[0]
+
+          # client_info.yaml contains information used to translate client ID
+          # to FQDN; extract it and save for later
```

Comment:
Please add a comment here that this file is not extracted, but skipped.

Reply:
Done.